server/repository/src/migrations/v1_06_00/cold_chain_indexes.rs

```diff
@@ -0,0 +1,12 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+        CREATE INDEX index_temperature_log_datetime ON temperature_log (datetime);
```

Comment:
I think we would want indexes on references:

Btw I did this PR awhile ago: https://github.com/msupply-foundation/open-msupply/pull/971, there is sql statement in PR description, to check which relations are not indexed, I got output below. 

Do you think it's worth adding all of them ? I think at the very least the ones that are concerned with cold chain:

```sql
"CREATE INDEX ""i_name_store_join_name_link_id_fkey"" ON ""name_store_join"" (""name_link_id"");"
"CREATE INDEX ""i_temperature_breach_config_store_id_fkey"" ON ""temperature_breach_config"" (""store_id"");"
"CREATE INDEX ""i_invoice_line_inventory_adjustment_reason_id_fkey"" ON ""invoice_line"" (""inventory_adjustment_reason_id"");"
"CREATE INDEX ""i_invoice_line_item_link_id_fkey"" ON ""invoice_line"" (""item_link_id"");"
"CREATE INDEX ""i_document_form_schema_id_fkey"" ON ""document"" (""form_schema_id"");"
"CREATE INDEX ""i_document_owner_name_id_fkey"" ON ""document"" (""owner_name_id"");"
"CREATE INDEX ""i_document_context_id_fkey"" ON ""document"" (""context_id"");"
"CREATE INDEX ""i_contact_trace_program_id_fkey"" ON ""contact_trace"" (""program_id"");"
"CREATE INDEX ""i_contact_trace_document_id_fkey"" ON ""contact_trace"" (""document_id"");"
"CREATE INDEX ""i_contact_trace_patient_id_fkey"" ON ""contact_trace"" (""patient_id"");"
"CREATE INDEX ""i_contact_trace_contact_patient_id_fkey"" ON ""contact_trace"" (""contact_patient_id"");"
"CREATE INDEX ""i_contact_trace_store_id_fkey"" ON ""contact_trace"" (""store_id"");"
"CREATE INDEX ""i_stock_line_supplier_id_fkey"" ON ""stock_line"" (""supplier_id"");"
"CREATE INDEX ""i_stock_line_barcode_id_fkey"" ON ""stock_line"" (""barcode_id"");"
"CREATE INDEX ""i_sensor_store_id_fkey"" ON ""sensor"" (""store_id"");"
"CREATE INDEX ""i_sensor_location_id_fkey"" ON ""sensor"" (""location_id"");"
"CREATE INDEX ""i_barcode_item_id_fkey"" ON ""barcode"" (""item_id"");"
"CREATE INDEX ""i_temperature_log_sensor_id_fkey"" ON ""temperature_log"" (""sensor_id"");"
"CREATE INDEX ""i_temperature_log_store_id_fkey"" ON ""temperature_log"" (""store_id"");"
"CREATE INDEX ""i_temperature_log_location_id_fkey"" ON ""temperature_log"" (""location_id"");"
"CREATE INDEX ""i_temperature_log_temperature_breach_id_fkey"" ON ""temperature_log"" (""temperature_breach_id"");"
"CREATE INDEX ""i_encounter_clinician_id_fkey"" ON ""encounter"" (""clinician_id"");"
"CREATE INDEX ""i_encounter_enrolment_program_id_fkey"" ON ""encounter"" (""program_id"");"
"CREATE INDEX ""i_program_event_context_id_fkey"" ON ""program_event"" (""context_id"");"
"CREATE INDEX ""i_program_master_list_id_fkey"" ON ""program"" (""master_list_id"");"
"CREATE INDEX ""i_program_context_id_fkey"" ON ""program"" (""context_id"");"
"CREATE INDEX ""i_program_requisition_order_typ_program_requisition_settings_fkey"" ON ""program_requisition_order_type"" (""program_requisition_settings_id"");"
"CREATE INDEX ""i_requisition_line_item_link_id_fkey"" ON ""requisition_line"" (""item_link_id"");"
"CREATE INDEX ""i_name_tag_join_name_tag_id_fkey"" ON ""name_tag_join"" (""name_tag_id"");"
"CREATE INDEX ""i_name_tag_join_name_link_id_fkey"" ON ""name_tag_join"" (""name_link_id"");"
"CREATE INDEX ""i_report_argument_schema_id_fkey"" ON ""report"" (""argument_schema_id"");"
"CREATE INDEX ""i_temperature_breach_sensor_id_fkey"" ON ""temperature_breach"" (""sensor_id"");"
"CREATE INDEX ""i_temperature_breach_store_id_fkey"" ON ""temperature_breach"" (""store_id"");"
"CREATE INDEX ""i_temperature_breach_location_id_fkey"" ON ""temperature_breach"" (""location_id"");"
"CREATE INDEX ""i_user_permission_context_id_fkey"" ON ""user_permission"" (""context_id"");"
"CREATE INDEX ""i_requisition_period_id_fkey"" ON ""requisition"" (""period_id"");"
"CREATE INDEX ""i_requisition_name_link_id_fkey"" ON ""requisition"" (""name_link_id"");"
"CREATE INDEX ""i_period_period_schedule_id_fkey"" ON ""period"" (""period_schedule_id"");"
"CREATE INDEX ""i_document_registry_form_schema_id_fkey"" ON ""document_registry"" (""form_schema_id"");"
"CREATE INDEX ""i_document_registry_context_id_fkey"" ON ""document_registry"" (""context_id"");"
"CREATE INDEX ""i_program_requisition_settings_name_tag_id_fkey"" ON ""program_requisition_settings"" (""name_tag_id"");"
"CREATE INDEX ""i_program_requisition_settings_program_id_fkey"" ON ""program_requisition_settings"" (""program_id"");"
"CREATE INDEX ""i_program_requisition_settings_period_schedule_id_fkey"" ON ""program_requisition_settings"" (""period_schedule_id"");"
"CREATE INDEX ""i_invoice_clinician_id_fkey"" ON ""invoice"" (""clinician_id"");"
"CREATE INDEX ""i_invoice_name_link_id_fkey"" ON ""invoice"" (""name_link_id"");"
"CREATE INDEX ""i_plugin_data_store_id_fkey"" ON ""plugin_data"" (""store_id"");"
"CREATE INDEX ""i_program_enrolment_program_id_fkey"" ON ""program_enrolment"" (""program_id"");"
"CREATE INDEX ""i_master_list_name_join_name_link_id_fkey"" ON ""master_list_name_join"" (""name_link_id"");"
"CREATE INDEX ""i_master_list_line_item_link_id_fkey"" ON ""master_list_line"" (""item_link_id"");"
"CREATE INDEX ""i_clinician_store_join_clinician_id_fkey"" ON ""clinician_store_join"" (""clinician_id"");"
"CREATE INDEX ""i_clinician_store_join_store_id_fkey"" ON ""clinician_store_join"" (""store_id"");"
"CREATE INDEX ""i_stocktake_line_inventory_adjustment_reason_id_fkey"" ON ""stocktake_line"" (""inventory_adjustment_reason_id"");"
"CREATE INDEX ""i_stocktake_line_item_link_id_fkey"" ON ""stocktake_line"" (""item_link_id"");"
```

Reply:
yeah, I think we should - even though they aren't useful for the aggregation query (in my testing postgres didn't use any other indexes which I tried). 
I've removed the `_link` tables, because those aren't in develop yet and renamed from cold_chain_indexes

🙏